app/models/orphan.rb

```diff
@@ -134,11 +136,15 @@ def sponsored_siblings_does_not_exceed_siblings_count
   end
 
   def default_sponsorship_status_unsponsored
-    self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    if new_record?
+      self.orphan_sponsorship_status ||= OrphanSponsorshipStatus.find_by_name 'Unsponsored'
+    end
   end
 
   def default_orphan_status_active
-     self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    if new_record?
+      self.orphan_status ||= OrphanStatus.find_by_name 'Active'
+    end
```

Comment:
One could argue that the single responsibility is initializing an instance in that case